app/views/posts/index.html.erb

```diff
@@ -0,0 +1,15 @@
+<% provide(:title, "Comments") %>
+<h1>Comment</h1>
+
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+  <% @posts.each do |p|%>
+    <ul>
+    <li> Comment: <%= p.body %>,  </li>
+    <% if @logged_in %>
+     Written by:    <%= p.user.name %>,
+    <% end %>
```

Comment:
[OPTIONAL]
You should have made `logged_in?' a helper so that way you just use it directly here in the view.